owasp-suppressions.xml

```diff
@@ -101,4 +102,8 @@
     <packageUrl regex="true">^pkg:maven/io\.netty/netty.*@.*$</packageUrl>
     <vulnerabilityName>CVE-2023-4586</vulnerabilityName>
   </suppress>
+  <suppress until="2023-12-30Z">
```

Comment:
Please fix this rather than suppressing

Reply:
Was suppressed because there is no mitigation currently available.
https://access.redhat.com/security/cve/CVE-2023-4586

Reply:
4586 is the suppression above this one. You're correct that one can be suppressed as described in https://github.com/hypertrace/hypertrace-bom/blob/6e9d9aa4f9a9631c281df4b4c8916490d5cd39d9/owasp-suppressions.xml#L29-L40

The issue here is the newly added one, [44487](https://nvd.nist.gov/vuln/detail/CVE-2023-44487) - that is the HTTP2 Rapid RST vuln and should not be suppressed. Netty`*.100` contains the fix, and our latest framework libs should already include that if upgraded.